README.md

```diff
@@ -65,6 +65,8 @@ Then install the COVID ML Profile, which is a portable profile containing all FK
 
 After the profile installed, wait until all PODs ready and FK-Covid can be accessed via URLs retrieved from `kubectl get svc -n kubeflow`.
 
+After accessed to Minio browser and login page asks you to fill in `Access Key` and `Secret Key`, they're hardcoded as `minio` and `minio123`.
```

Comment:
Could you please rephrase it to this?

When opening browser to the MinIO page, and the login page will ask you to fill in `Access Key` and `Secret Key`. Please use the default keys as `minio` and `minio123`.

Reply:
Sure, no problem, let me rephrase it.